sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/AdministrationOperationsLiveTests.cs

```diff
@@ -218,33 +181,5 @@ private void ValidateDocumentModelDetails(DocumentModelDetails model, string des
 
             // TODO add validation for Doctypes https://github.com/Azure/azure-sdk-for-net-pr/issues/1432
         }
-
-        private void ValidateDocumentModelSummary(DocumentModelSummary model, string description = null, IReadOnlyDictionary<string, string> tags = null)
```

Comment:
This helper method was only used by the removed `AdminOps` test, so there's no need to keep it around anymore.